recipe/symfony.php

```diff
@@ -62,6 +62,13 @@
     }
 });
 
+desc('Optimize environment variables');
+task('deploy:env', function () {
+    within('{{release_or_current_path}}', function () {
+        run('{{bin/composer}} dump-env "${APP_ENV:-prod}"');
+    });
+});
+
 desc('Deploys project');
 task('deploy', [
```

Comment:
If you want to dump env on every deployment, you should add it to the `deploy` task?